src/components/Footer/__test__/Footer.test.tsx

```diff
@@ -37,7 +37,11 @@ describe("Footer Link Redirection", () => {
     await fireEvent.click(contactLink)
 
     act(()=>{
-        render(<Contact></Contact>)
+        render(
+          <MemoryRouter>
+            <Contact></Contact>
```

Comment:
```suggestion
            <Contact/>
```
use component shorthand